unit_tests/test_base_version.py

```diff
@@ -126,7 +126,7 @@ def test_2024_2(self):
         scylla_repo = self.url_base + '-enterprise/enterprise-2024.2/rpm/centos/latest/scylla.repo'
         linux_distro = 'centos-9'
         version_list = general_test(scylla_repo, linux_distro)
-        assert set(version_list) == {'6.0', '2024.1'}
```

Comment:
please don't introduce this change, it's wrong, we don't have a release of 2024.2 yet.

Reply:
it failed unit test

Reply:
I know, and as you can see I've fixed it.

Reply:
yes, saw it later and rebased my PR